loader/loader_linux.c

```diff
@@ -330,12 +330,20 @@ VkResult linux_read_sorted_physical_devices(struct loader_instance *inst, uint32
 
     // Add all others after (they've already been sorted)
     for (uint32_t dev = 0; dev < phys_dev_count; ++dev) {
+        loader_set_dispatch((void *)sorted_device_term[dev], inst->disp);
         sorted_device_term[dev]->this_icd_term = sorted_device_info[dev].icd_term;
         sorted_device_term[dev]->icd_index = sorted_device_info[dev].icd_index;
         sorted_device_term[dev]->phys_dev = sorted_device_info[dev].physical_device;
-        loader_set_dispatch((void *)sorted_device_term[dev], inst->disp);
+
         loader_log(inst, VULKAN_LOADER_INFO_BIT | VULKAN_LOADER_DRIVER_BIT, 0, "           [%u] %s  %s", dev,
                    sorted_device_info[dev].device_name, (sorted_device_info[dev].default_device ? "[default]" : ""));
+
+        // Fill in the properties
+        sorted_device_term[dev]->this_icd_term->dispatch.GetPhysicalDeviceProperties(sorted_device_term[dev]->phys_dev,
+                                                                                     &sorted_device_term[dev]->properties);
+
+        // Fill in driver support of loader-interested device extensions
+        device_extensions_supported_by_physical_device(sorted_device_term[dev]);
```

Comment:
Is this suppose to be applied to *all* physical devices? If so then this needs to be done int its own loop, possibly as a cleanup to this function (all the common 'setup' to be done in its own loop). Or not, as this PR is MASSIVE.